### PR TITLE
[DECLUTTER FAQ] Add Backup and Recovery

### PR DESCRIPTION
**Pull request by RiccardoMasutti**
I created a new chapter, moved some FAQs and added new ones.

### CODE REVIEW

**Comment by yahiheb** [comment]
I am not sure about adding all the reddit questions to the FAQs.
We have been doing this before but I think we should stop doing that.

**Comment by RiccardoMasutti** [comment]
> I am not sure about adding all the reddit questions to the FAQs.
> We have been doing this before but I think we should stop doing that.

@MaxHillebrand What do you think?

**Comment by yahiheb** [comment]
> @yahiheb I didn't even know that there were FAQs in reddit... I am in favor of having it all in the docs

I am not saying there are FAQs in Wasabi sub-reddit, what I am saying is we shouldn't add all the questions that get asked in reddit to the docs, otherwise we will have hundreds of questions which doesn't make sense.
FAQs stands for frequently asked questions not all the questions, and what lead us to this decluttering is that we add a lot of repeated information and some times not accurate or not clear info.

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -1032,6 +989,69 @@ Wasabi Wallet will use only the necessary coins to CoinJoin (up to 7 coins).
...
+Having a close guess of what the password is increases the possibility that you get your password.
+:::
+
+::::details
+### I lost seed words but still have .json file and the password. Is my wallet still recoverable?
+
+Yes, the .json file contains your private keys in it.
+That gives you access to your wallet in combination with the passphrase that you still have.
+
+In this case it is still advisable to create a new wallet from scratch and save the new seed + password.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md]:
```suggestion
No.
The password you set is used as a 13th word (passphrase) as described in BIP39, you should back it up when you generate a wallet.
It is necessary to spend your bitcoin or to recover your wallet.
:::

:::details
### What should I do if I forget my password?

Try to use the [Password Finder Tool](/using-wasabi/PasswordFinder.md) to find your password.
Having a close guess of what the password is increases the possibility that you get your password.
:::

:::details
### I lost the recovery words but I still have the .json file and the password. Is my wallet still recoverable?

Yes, but in this case it is advisable to create a new wallet and back up the new recovery words and the password, then move your bitcoin there.

The .json file contains the encrypted secret that requires the password to derive your private keys.
That gives you access to your bitcoin.
```

**Pull request merged**